examples/tangent_pca_so3.py

```diff
@@ -29,10 +31,10 @@ def main():
     tpca = TangentPCA(metric=METRIC, n_components=N_COMPONENTS)
     tpca = tpca.fit(data, base_point=mean_estimate)
     tangent_projected_data = tpca.transform(data)
-    print(
+    logging.info(
         'Coordinates of the Log of the first 5 data points at the mean, '
         'projected on the principal components:')
-    print(tangent_projected_data[:5])
+    logging.info('\n{}'.format(tangent_projected_data[:5]))
```

Comment:
same question

Reply:
My reasoning for this case is the same as `examples/tangent_pca_s2.py `

Reply:
Ok!